tests/jeos/grub2_gfxmode.pm

```diff
@@ -12,13 +12,16 @@
 
 use Mojo::Base qw(opensusebasetest);
 use testapi;
-use serial_terminal 'select_serial_terminal';
+use serial_terminal;
 use jeos qw(set_grub_gfxmode);
+use utils qw(ensure_serialdev_permissions);
 
 sub run {
     select_serial_terminal;
 
     set_grub_gfxmode;
+    ensure_serialdev_permissions;
```

Comment:
This is a bit out of place here IMO

Does this work in all scenarios?

Reply:
It has worked here http://kepler.suse.cz/tests/22594#step/grub2_gfxmode/32 despite that it was done in first_run test module as well. I have rather put this in a test module than into combustion

Reply:
If firstrun already performs `ensure_serialdev_permissions` it shouldn't be necessary to do it again. Maybe the `dialout` group is missing or something else breaks it?

Reply:
Right, now I get it. Arguably whatever creates the user is responsible for calling `ensure_serialdev_permissions` and grub2_gfxmode is not the right place. If doing it in combustion is not an option, `verify_setup` or `system_prepare` might fit

Reply:
That is true, I want to turn `grub2_gfxmode` module into something like https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/transactional/host_config.pm but for minimal-vm hosts. I haven't just renamed the module in this PR

Reply:
And verification module should not do system-wide changes IMHO